Symfony/CS/Fixer/Contrib/PhpdocToCommentFixer.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Ceeram <ceeram@cakephp.org>
+ */
+class PhpdocToCommentFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $index => $token) {
+            $nextIndex = $tokens->getNextMeaningfulToken($index);
+
+            // skip if there is no next token or if next token is block end `}`
+            if (null === $nextIndex || $tokens[$nextIndex]->equals('}')) {
+                continue;
+            }
+
+            if ($this->isStructuralElement($tokens[$nextIndex])) {
+                continue;
+            }
+
+            if ($tokens[$nextIndex]->isGivenkind(T_FOREACH) && $this->isValidForeach($tokens, $index)) {
+                continue;
+            }
+
+            $token->override(array(T_COMMENT, '/*'.ltrim($token->getContent(), '/*'), $token->getLine()));
+        }
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'Docblocks should only be used on structural elements.';
+    }
+
+    /**
+     * Check if token is a structural element
+     * @see http://www.phpdoc.org/docs/latest/glossary.html#term-structural-elements
+     *
+     * @param \Symfony\CS\Tokenizer\Token $token
+     * @return bool
+     */
+    private function isStructuralElement(Token $token)
+    {
+        $skip = array(
+            T_PRIVATE,
+            T_PROTECTED,
+            T_PUBLIC,
+            T_FUNCTION,
+            T_ABSTRACT,
+            T_CONST,
+            T_NAMESPACE,
+            T_REQUIRE,
+            T_REQUIRE_ONCE,
+            T_INCLUDE,
+            T_INCLUDE_ONCE,
+        );
+
+        return $token->isClassy() || $token->isGivenKind($skip);
+    }
+
+    /**
+     * Checks foreach statements for correct docblock usage.
+     *
+     * @param \Symfony\CS\Tokenizer\Tokens $tokens
+     * @param int $index
+     * @return bool
```

Comment:
please keep an empty phpdoc line between params and returns (the automatic fixer for this is not merged yet). This will also fix the test error about param alignment (currently, the are in the same group)